web/libs/editor/src/components/App/App.jsx

```diff
@@ -103,9 +103,11 @@ class App extends Component {
       >
         <Result status="success" title={getEnv(this.props.store).messages.NO_NEXT_TASK} />
         <Block name="sub__result">You have completed all tasks in the queue!</Block>
-        <Button onClick={(e) => store.prevTask(e, true)} look="outlined" style={{ margin: "16px 0" }}>
-          Go to Previous Task
-        </Button>
+        {store.taskHistory.length > 0 && (
```

Comment:
oh, thank you, it was pretty annoying for me, didn't think the fix is so simple!